forum/qa-plugin/q2apro-on-site-notifications/styles.css

```diff
@@ -49,6 +49,7 @@
     padding: 3px 0px 12px 0;
     border-bottom: 1px dashed #CCC;
 	text-align: left;
+	transition: 350ms all !important;

```

Comment:
czy te importanty są konieczne?

Reply:
Przynajmniej aby działało, jeżeli znasz inne rozwiązanie (na ten moment) to napisz

Reply:
Co nadpisuje style, którym nadałeś `!important`? Z tego co widzę, to tranzycję wszędzie dodałeś taką samą **350ms all**. Załóżmy, że w przyszłości będzie potrzeba zmiany czasu trwania - wtedy w każdym pojedynczym stylu trzeba będzie to zaktualizować. Zamiast tego, utwórz jedną klasę z nazwą np. **animateAllProperties**, w niej wstaw `transition: 350ms all;` i klasę CSSową przypisz do elementów HTML, której ona dotyczy. Wtedy w razie potrzeby zmiany wystarczy zaktualizować wartość w jednym miejscu. Powinno wtedy obejść się bez używania `!important`, a jeśli jakiś styl będzie nadpisywał tranzycję z klasy **animateAllProperties**, to do jej selektora dopiszesz selektor danego elementu, żeby podbić mu specyficzność.